app/repositories/apps.py

```diff
@@ -28,3 +39,52 @@ async def get_app_status_sub():
         yield json.dumps(app_list[i - 1])
         await asyncio.sleep(4)
         switch = not switch
+
+
+async def install_app_sub(app_id: str):
+    if(not app_id in available_app_ids):
+        raise HTTPException(
+            status.HTTP_400_BAD_REQUEST, detail="script does not exist"
+        )
+
+    await send_sse_message(SSE.INSTALL_APP, {"id": app_id})
+    loop = asyncio.get_event_loop()
+    loop.create_task(run_bonus_script(app_id, "on"))
+
+    return jsonable_encoder({"id": app_id})
+
+
+async def uninstall_app_sub(app_id: str):
+    if(not app_id in available_app_ids):
+        raise HTTPException(
+            status.HTTP_400_BAD_REQUEST, detail="script does not exist"
+        )
+
+    await send_sse_message(SSE.INSTALL_APP, {"id": app_id})
+    loop = asyncio.get_event_loop()
+    loop.create_task(run_bonus_script(app_id, "off"))
+
+    return jsonable_encoder({"id": app_id})
+
+
+async def run_bonus_script(app_id: str, params: str):
+    script_path = f"{SHELL_SCRIPT_PATH}config.scripts/bonus.{app_id}.sh"
+    print(script_path)
+    cmd = f"bash {script_path} {params}"
+
+    proc = await asyncio.create_subprocess_shell(
+        cmd,
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.PIPE,
+    )
+
+    stdout, stderr = await proc.communicate()
+
+    print(f'[{cmd!r} exited with {proc.returncode}]')
+    if stdout:
+        print(f'[stdout]\n{stdout.decode()}')
+    if stderr:
+        print(f'[stderr]\n{stderr.decode()}')
+
```

Comment:
I've started to use `logging.log()` for all logging related things. Something to keep in mind for next PR's :)